homeassistant/components/sonos/media_player.py

```diff
@@ -30,6 +30,7 @@
     ATTR_NIGHT_SOUND, ATTR_SLEEP_TIME, ATTR_SPEECH_ENHANCE, ATTR_TIME,
     ATTR_VOLUME, ATTR_WITH_GROUP,
     SERVICE_CLEAR_TIMER, SERVICE_JOIN, SERVICE_RESTORE, SERVICE_SET_OPTION,
+    SERVICE_PLAY_QUEUE,
```

Comment:
It seems like service names are alphabetized 🔤 so please insert it after `SERVICE_JOIN` and reformat accordingly.